### DISCUSSION TITLE
fail loading clang from clang.framework….

### DISCUSSION THREAD

**Discussion by clint456** [thebaselab/codeapp#909] [Q&A] [answered]
![image](https://github.com/thebaselab/codeapp/assets/83180587/a5452561-6da9-4693-8544-17bcde84b3d4)

[this bug from one year ago](https://github.com/thebaselab/codeapp/issues/388)appears  again~ 
how to fix it QWQ

**Comment by bummoblizard** [comment, marked as answer]
Hi. Restarting your iPad should help. If that doesn't work, re-install the app.

**Reply by clint456** [reply]
ohhhhhh~ thanks, it works🤓
